### ISSUE TITLE
Feature Request: Map File/Multifile link list items to some field from returned pages

### ISSUE THREAD

**Issue by frankreporting** [mdelobelle/metadatamenu#110]
First, this plugin is super handy, and already one that I use every day! 

I have a multifile link field that gives me a list of folder notes, based on a tag `#area`. So I have:

`dv.pages("#area")`

Which returns a list like...

```
20 Personal
21 Automobile
22 etc..." 
```

It would be great if we could access the page object and map it, like we're rendering with the dataviewjs `dv.table...map()` or `dv.list...map()`. In my case, this would allow me to render the list using the aliases, so my dropdown list looks more like this:

```
🗃.Personal
🗃.Personal.Automobile
🗃.Personal.etc.
```

Nitpicky use case, I know, but I imagine others might find use in being able to render the list with something other than the filename.

**Comment by mdelobelle** [comment]
this is a nice idea indeed.
If we replace the link with a new string thanks to some kind of template, the plugin won't be able to identify the "current values" when you'll try to update the list.
What I would suggest is that the "custom rendering" goes into the alias of the link
So in your example it would translate into:

[[20 Personal|🗃.Personal]] which would be licely displayed in live preview and reading mode to 🗃.Personal

This way the link still exist and the plugin is able to find the current values in the field

WDYT?

**Comment by frankreporting** [comment]
> What I would suggest is that the "custom rendering" goes into the alias of the link
> 
> So in your example it would translate into:
> 
> 
> 
> [[20 Personal|🗃.Personal]] which would be licely displayed in live preview and reading mode to 🗃.Personal
> 
> 
> 
> This way the link still exist and the plugin is able to find the current values in the field
> 
> 
> 
> WDYT?

I like! Does that mean the pop-up  chooser menu would also render the list of values using their aliases? Or would the alias only appear in the note itself?

**Comment by mdelobelle** [comment]
I think would be the alias in case a custom rendering is set

**Comment by mdelobelle** [comment]
something like that ?

<img width="432" alt="image" src="https://user-images.githubusercontent.com/1233240/193824411-61bef38c-ce61-420f-89f8-c33315e4c6d3.png">

<img width="611" alt="image" src="https://user-images.githubusercontent.com/1233240/193824976-8972bcf8-a35b-4164-ab90-284f30c84dbc.png">

<img width="331" alt="image" src="https://user-images.githubusercontent.com/1233240/193825056-77016bad-b24b-4a61-ace0-5e6eebb3cc02.png">

**Comment by frankreporting** [comment]
Yes, that looks amazing!

**Comment by frankreporting** [comment]
Thank you for making that update! That's really slick! One minor thing: the autocomplete doesn't seem to work with the chosen alias, so you still have to search by original filename regardless of your chosen alias. Is that something that could be implemented in a future update? Would it help to have it as a separate feature request?

Thanks again for being responsive. Do you have a donation button or "buy me a coffee" set up? Would love to support your work!

**Comment by mdelobelle** [comment]
> Do you have a donation button

Not at the moment 😇, I do this purely for fun :) (and also for my personal needs)

> the autocomplete doesn't seem to work with the chosen alias

This is now available in 0.2.10
